backend/icpy/services/chat_service.py

```
@@ -189,7 +202,82 @@ def __init__(self, db_path: str = "chat.db"):
+                kind = item.get('kind') or item.get('type') or ''
+
+                # Infer kind from mime if not provided
+                if not kind:
+                    if isinstance(mime, str) and mime.startswith('image/'):
+                        kind = 'images'
+                    elif isinstance(mime, str) and mime.startswith('audio/'):
+                        kind = 'audio'
+                    else:
+                        kind = 'files'
+                # Map UI kind names to backend folder names
+                if kind in ('image', 'img'): kind = 'images'
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)